pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -251,7 +251,7 @@ def check_set_errors(self):
     def ask(self, command, query_delay=0):
         """Send a command and read the response, stripping white spaces.
 
-        Usually the properties use the `values` method that adds a strip call,
+        Usually the properties use the ``values()`` method that adds a strip call,

```

Comment:
```suggestion
        Usually the properties use the :meth:`values` method that adds a strip call,

```
With that, sphinx generates a link and also adds the parenthesis.

Reply:
This required a link to `CommonBase` in order to work, but that is in the last commit.